datastream-server/src/main/java/com/linkedin/datastream/server/providers/FileBasedPartitionThroughputProvider.java

```diff
@@ -49,6 +51,11 @@ public ClusterThroughputInfo getThroughputInfo(String clusterName) {
     return readThroughputInfoFromFile(partitionThroughputFile, clusterName);
   }
 
+  @Override
+  public ClusterThroughputInfo getThroughputInfo(DatastreamGroup datastreamGroup) {
```

Comment:
Setting this to NotImplemented means, any test added in LoadBasedPartitionAssignmentStrategy is not using the throughput information. Since, this will throw exception and the strategy after retry will switch too default mode.

Reply:
The tests for assignment logic are in a separate file. Strategy tests don't use `FileBasedPartitionThroughputProvider` and instead use a mock when necessary.